frontend/src/components/account/EditAccountPage.js

```diff
@@ -919,7 +923,9 @@ export default function EditAccountPage({
         {deleteEmail && (
           <Typography variant="subtitle2" className={classes.deleteMessage}>
             <InfoOutlinedIcon />
-            {texts.if_you_wish_to_delete} {deleteEmail}
+            {texts.if_you_wish_to_delete}
+            <div className={classes.spaceStrings}> </div>
+            <Link href="mailto:contact@climateconnect.earth">{deleteEmail}</Link>
```

Comment:
This needs to be generic otherwise the generic `deleteEmail` variable is pointless.
```suggestion
            <Link href={`mailto:${deleteEmail}`}>{deleteEmail}</Link>
```